package.json

```diff
@@ -30,7 +30,9 @@
   },
   "devDependencies": {
     "mocha": "1.x.x",
-    "chai": "1.x.x"
+    "chai": "1.x.x",
+    "@types/passport": "*",
```

Comment:
we should probably fix the versions to prevent possible problems like total change of the API of the library in the future (I know it almost doesn't happen... :D )